src/components/CustomerDetails/CollectCustomerDetailsScreen.tsx

```diff
@@ -219,7 +224,17 @@ const CollectCustomerDetailsScreen: FunctionComponent<NavigationFocusInjectedPro
   const getInputComponent = (): JSX.Element => {
     return selectedIdType.label === "Passport" &&
       selectedIdType.scannerType === "NONE" ? (
-      <ManualPassportInput
+      <PassportInputSection
+        hasScanner={false}
+        openCamera={() => null}
+        setIdInput={setIdInput}
+        submitId={() => onCheck(idInput)}
+      />
+    ) : selectedIdType.label === "Passport" &&
+      selectedIdType.scannerType === "QR" ? (
+      <PassportInputSection
+        hasScanner={true}
+        openCamera={() => setShouldShowCamera(true)}
         setIdInput={setIdInput}
         submitId={() => onCheck(idInput)}
       />
```

Comment:
What do you think about passing `selectedIdType.scannerType` down to the component instead? You can remove the duplication of `PassportInputSection` here by doing that.

```suggestion
      <PassportInputSection
        scannerType={selectedIdType.scannerType}
        openCamera={() => setShouldShowCamera(true)}
        setIdInput={setIdInput}
        submitId={() => onCheck(idInput)}
      />
```